lib/config.js

```diff
@@ -61,6 +62,16 @@ function loadConfig(filePath) {
         }
     }
 
+    // Recursively merge parent configurations
+    if (config.extends) {
+        var parentPath = (!isAbsolutePath(config.extends) ?
+            path.join(path.dirname(filePath), config.extends) :
+            config.extends
+        );
+
+        config = util.mergeConfigs(loadConfig(parentPath), config);
```

Comment:
I suppose you mean in case the `extends` file specified could not be found/parsed?
Could do that, but since it's recursive it's a bit tricky. What do we want the output to be?

We could, for instance, add the source configs to the message, so the output would be something like:

```
Cannot read config file: /home/espenh/webdev/eslint/tests/fixtures/config-extends/subdir/subsubdir/subsubsubdir/deepesta.json
Error: ENOENT: no such file or directory, open '/home/espenh/webdev/eslint/tests/fixtures/config-extends/subdir/subsubdir/subsubsubdir/non-existing-file.json'
Referenced from: /home/espenh/webdev/eslint/tests/fixtures/config-extends/subdir/subsubdir/deeper.json
Referenced from: /home/espenh/webdev/eslint/tests/fixtures/config-extends/deep.json
```
Or possibly something a little prettier/better. Suggestions?